src/cascadia/TerminalApp/Jumplist.cpp

```diff
@@ -118,8 +118,9 @@ static std::wstring_view _getExePath()
 // - settings - The settings object to update the jumplist with.
 // Return Value:
 // - <none>
-HRESULT Jumplist::UpdateJumplist(const CascadiaSettings& settings) noexcept
+winrt::fire_and_forget Jumplist::UpdateJumplist(const CascadiaSettings& settings) noexcept
 {

```

Comment:
we might want to grab a copy of `settings` and operate on it inside the coroutine ... otherwise, this reference will 100% be invalid by the time we resume the thread.

Reply:
Fixed